File: Java/Assignment 10/TestingSystem_Assignment_10/src/com/vti/entity/Account.java
package com.vti.entity;

import java.time.LocalDate;


public class Account {
	private int id;
	private String email;
	private String username;
	private String fullName;
	private Department department;
	private Position position;
	private LocalDate createDate;

	/**
	 * @return the id
	 */
	public int getId() {
		return id;
	}

	/**
	 * @return the email
	 */
	public String getEmail() {
		return email;
	}

	/**
	 * @return the department
	 */
	public String getDepartment() {
		return department.getName();
	}

	/**
	 * @return the position
	 */

	/**
	 * @param id the id to set
	 */
	public void setId(int id) {
		this.id = id;
	}

	/**
	 * @return the position
	 */
	public String getPosition() {
		return position.getName();
	}

	/**
	 * @param position the position to set
	 */
	public void setPosition(Position position) {
		this.position = position;
	}

	public void setID(int iD) {
		id = iD;
	}

	/**
	 * @return the createDate
	 */
	public LocalDate getCreateDate() {
		return createDate;
	}

	/**
	 * @param createDate the createDate to set
	 */
	public void setCreateDate(LocalDate createDate) {
		this.createDate = createDate;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	/**
	 * @return the username
	 */
	public String getUsername() {
		return username;
	}

	/**
	 * @param username the username to set
	 */
	public void setUsername(String username) {
		this.username = username;
	}

	/**
	 * @return the fullName
	 */
	public String getFullName() {
		return fullName;
	}

	/**
	 * @param fullName the fullName to set
	 */
	public void setFullName(String fullName) {
		this.fullName = fullName;
	}

	public void setDepartment(Department department) {
		this.department = department;
	}

	@Override
	public String toString() {
		return "Account [ID=" + id + ", email=" + email + ", Username=" + username + ", FullName=" + fullName
				+ ", department=" + department + ", position=" + position + ", CreateDate=" + createDate + "]";
	}

}

File: ReactJS/TypeScript/LAB2_TypeScript/Account.js
"use strict";
exports.__esModule = true;
exports.printInfoAccount = exports.Account = void 0;
var Account = /** @class */ (function () {
    function Account(id, email, username, fullName, department, position, createDate) {
        this.id = id;
        this.email = email;
        this.username = username;
        this.fullName = fullName;
        this.department = department;
        this.position = position;
        this.createDate = createDate;
    }
    Account.prototype.getId = function () {
        return this.id;
    };
    Account.prototype.getEmail = function () {
        return this.email;
    };
    Account.prototype.getUsername = function () {
        return this.username;
    };
    Account.prototype.getFullName = function () {
        return this.fullName;
    };
    Account.prototype.getDepartment = function () {
        return this.department;
    };
    Account.prototype.getPosition = function () {
        return this.position;
    };
    Account.prototype.getCreateDate = function () {
        return this.createDate;
    };
    Account.prototype.setId = function (value) {
        this.id = value;
    };
    Account.prototype.setEmail = function (value) {
        this.email = value;
    };
    Account.prototype.setUsername = function (value) {
        this.username = value;
    };
    Account.prototype.setFullName = function (value) {
        this.fullName = value;
    };
    Account.prototype.setDepartment = function (value) {
        this.department = value;
    };
    Account.prototype.setPosition = function (value) {
        this.position = value;
    };
    Account.prototype.setCreateDate = function (value) {
        this.createDate = value;
    };
    return Account;
}());
exports.Account = Account;
function printInfoAccount(account) {
    console.log("ID: " +
        account.getId() +
        " Email: " +
        account.getEmail() +
        " Username: " +
        account.getUsername() +
        " FullName: " +
        account.getFullName() +
        " Department: " +
        account.getDepartment().getName() +
        " Position: " +
        account.getPosition().getName() +
        " Create Date: " +
        account.getCreateDate());
}
exports.printInfoAccount = printInfoAccount;
